README.md

```diff
@@ -145,6 +145,37 @@ scalingo.osc_fr1.apps.all # OR scalingo.region(:osc_fr1).apps.all
 scalingo.apps.create(name: "my-new-app", dry_run: true)
 ```
 
+### Interacting with databases
+
+Requests to the [database API](https://developers.scalingo.com/databases/) requires
+extra authentication for each addon you want to interact with. [Addon authentication
+tokens are valid for one hour](https://developers.scalingo.com/addons#get-addon-token).
+
+Supported regions for database API are `db_api_osc_fr1` and `db_api_osc_secnum_fr1`.
+
+```ruby
+require "scalingo"
+
+scalingo = Scalingo::Client.new
+scalingo.authenticate_with(access_token: "my_access_token")
+
+# First, authenticate using the `addons` API
+scalingo.osc_fr1.addons.authenticate(app_id, addon_id)
+
+# Once authenticated for that specific addon, you can interact with
+# database and backup APIs.
+# IDs of databases are the IDs of the corresponding addons
+
+# get all information for a given database
+scalingo.db_api_osc_fr1.databases.find(addon_id)
```

Comment:
What about automating the authentication when hitting the database API?

```suggestion
scalingo.db_api_osc_fr1.databases.find(app_id, addon_id)
```


I do not like that much the stateful 

```
scalingo.osc_fr1.addons.authenticate(app_id, addon_id)
```

I would be in favor of keeping only 1 authentication method.

What do you thing?

Reply:
I thought about that at first indeed.

I noticed that at the moment, the gem seems to be a direct abstraction of the API without extra process. That is why I stick to it even for the authentication to the database api.

Automation of the authentication mechanism for the database API would result in far more logic behind the scene. It would totally break its current apparent paradigm.

I would be in favor of providing a first version as-is to get some feedback and eventually add some automation in a second time depending the demand. Such logic could be part of a distinct gem, or built within a distinct lib inside that one, or something else. IMO sticking to the API as a first step is more cautious and may buy us some time to think more on further steps.

Reply:
Thanks a lot for your feedback on this, I understand it. I'm adding @ksol  in the loop since he's been maintainer. If it's good to him it will be good to me

Reply:
oh, thanks for cracking this one open! at first glance it looks pretty solid. I'll take a closer look once I have more coffee than blood in my veins